glossary.yml

```diff
@@ -3360,7 +3360,7 @@
     term: "LaTeX"
     def: >
       A typesetting system for document preparation that uses a specialized [markup language](#markup_language) to define a document structure
-      (e.g. headings), stylise text, insert mathematical equations, and manage citations and cross-references. LaTeX is widely
+      (e.g., headings), stylize text, insert mathematical equations, and manage citations and cross-references. LaTeX is widely
```

Comment:
This is a British vs American spelling; I believe British is preferred, so this should not be changed.

Reply:
```suggestion
      (e.g., headings), stylise text, insert mathematical equations, and manage citations and cross-references. LaTeX is widely
```